src/components/SaveThresholdModal.vue

```diff
@@ -232,10 +232,10 @@ export default defineComponent({
           showToast(translate('Service has been scheduled'))
           this.closeModal();
         } else {
-          showToast(translate('Something went wrong'))
+          showToast(translate('Something went wrong'), getResponseError(ifScheduleJobSuccess))
```

Comment:
`ifScheduleJobSuccess` contains a boolean value and hence will not contain the server error.